### PR TITLE
Add post about using bundle option

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: content/posts/2024-03-22-use-bundle-option.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "How to use bundle option"
+date: 2024-03-22T12:39:10+05:30
+---
+
+CRC provides a config option called `bundle` to configure the _location_ of the bundle. This option is useful when you
+want to use a bundle that is not the default one or user want to try a bundle which is not part of release but available
+at OpenShift mirror location. The `bundle` option can accept local filesystem paths, container registry URLs, and
```
Comment (gbraad) [inline content/posts/2024-03-22-use-bundle-option.md]:
```patch
- at ... mirror
+ at the ... mirror
```

File: content/posts/2024-03-22-use-bundle-option.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Using `bundle` option with `crc setup` and `crc start` commands
+```bash
+crc setup --bundle https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/microshift/4.15.0/crc_microshift_libvirt_4.15.0_amd64.crcbundle
+crc start --bundle https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/microshift/4.15.0/crc_microshift_libvirt_4.15.0_amd64.crcbundle
+```
+
+## Conclusion
+This blog post has shown you how to use the `bundle` option with CRC. By following these steps, you can quickly and easily
+spin up a local cluster using the bundle of your choice.
+As usual, we'd like to get as much [feedback](https://github.com/crc-org/crc/issues/new/choose) as possible on all this work!
```
Comment (gbraad) [inline content/posts/2024-03-22-use-bundle-option.md]:
```patch
- as much
+ your
```

**Review by gbraad — CHANGES_REQUESTED** [review]
minor suggestions

File: content/posts/2024-03-22-use-bundle-option.md (hunk 1 above)
Comment (praveenkumar) [reply content/posts/2024-03-22-use-bundle-option.md]: Done

File: content/posts/2024-03-22-use-bundle-option.md (hunk 2 above)
Comment (praveenkumar) [reply content/posts/2024-03-22-use-bundle-option.md]: Done

**Comment by cfergeau** [comment]
Does this verify the bundle signature?

**Review by gbraad — APPROVED** [review]

**Comment by praveenkumar** [comment]
> Does this verify the bundle signature?

By default when use `-b` option we don't verify bundle signature. Signature only verified for default bundle which is part of specific crc release.

**Comment by cfergeau** [comment]
> > Does this verify the bundle signature?
> 
> By default when use `-b` option we don't verify bundle signature. Signature only verified for default bundle which is part of specific crc release.

This in my opinion is very important information to provide in the blog post, plus a way of verifying manually the signature of what was downloaded.

**Comment by praveenkumar** [comment]
> > > Does this verify the bundle signature?
> > 
> > 
> > By default when use `-b` option we don't verify bundle signature. Signature only verified for default bundle which is part of specific crc release.
> 
> This in my opinion is very important information to provide in the blog post, plus a way of verifying manually the signature of what was downloaded.

So you want user to download the bundle first using curl/wget/browser and then verify it with the signature and then use the bundle path instead directly using bundle url?

**Comment by cfergeau** [comment]
> > This in my opinion is very important information to provide in the blog post, plus a way of verifying manually the signature of what was downloaded.
> 
> So you want user to download the bundle first using curl/wget/browser and then verify it with the signature and then use the bundle path instead directly using bundle url?


This can take the form of 
> Beware, if you use -b https://// there will be no signature/checksum/... checks
> Bundles downloaded from mirror.openshift.com can be verified this way: 
> ...

**Comment by praveenkumar** [comment]
> > > This in my opinion is very important information to provide in the blog post, plus a way of verifying manually the signature of what was downloaded.
> > 
> > 
> > So you want user to download the bundle first using curl/wget/browser and then verify it with the signature and then use the bundle path instead directly using bundle url?
> 
> This can take the form of
> 
> > Beware, if you use -b https://// there will be no signature/checksum/... checks
> > Bundles downloaded from mirror.openshift.com can be verified this way:
> > ...

Updated.

**Pull request merged**